README.md

```diff
@@ -90,7 +90,7 @@ The supported `boot/iso/` sub-directories (in A–Z order) are:
 
 [//]: # (distro-list-end)
 
-Unpopulated directories turns its matching boot menu entry off,
+Unpopulated directories turns off the matching boot menu entry,
```

Comment:
Thanks! I still think "turn" needs to drop the "s" to be correct, though, right? And I would s/the/their/ too.

Reply:
The [latest commit](https://github.com/thias/glim/pull/119/commits/6bfd4fa0d0b94511424d5dbcc7e829eaee10c2c6) is as simple as I can make it.
Hope it all checks out.